frontend/src/components/atoms/TablePagination/style.css

```diff
@@ -0,0 +1,32 @@
+.table-pagination-container {
+    display: flex;
```

Comment:
css files are mainly with 2 spaces indentation in the project. We should introduce a linter for this.